packages/react-components/react-dialog/src/components/DialogContent/DialogContent.types.ts

```diff
@@ -1,7 +1,7 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import type { ComponentProps, NextComponentState, Slot } from '@fluentui/react-utilities';
 
 export type DialogContentSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<'div'>>;
```

Comment:
Will it explode if we will keep previous definition?

Reply:
Yes. the previous definition is simply wrong. `root` slot **should** be non nullable **and** declared as `required`. since from this PR and forward we're finally treating `root` as proper slot if we don't properly declare it as non nullable and `required` we'll start to see it as optional in the render method.